docs/pages/getting-started/overview.md

```diff
@@ -85,11 +83,10 @@ Infrastructure diversity is a reality for any enterprise. Whether you're running
 
 ##### **Alpha** - Service mesh adapters adapters for which we are seeking community-contributed support.
```

Comment:
While you are here, can you please fix the duplication of the word `adapters` in this line?